### DISCUSSION TITLE
Can't see any appliances not errors

### DISCUSSION THREAD

**Discussion by plyusnin** [ekutner/home-connect-hass#168] [Community Support]
I set up the developer account, installed the integration, put the keys into `configuration.yaml` and restarted HA. But after that, I can see nothing. I have only my Bosch wash dryer, so it should not be related to the limit of 3 devices.
I can see the washing machine in the original app without WiFi, the machine is ON.
Am I right that I expect the machine to automatically be added to the entities list with the name, containing "bosch"?
Also, I didn't get any pop-ups when the integration was installed (somewhere I can see that there should be a pop-up asking for the credentials).
Could anyone please help me and say what I am missing here? Thanks!

**Comment by ekutner** [comment]
This kind of problem is always the result of misconfiguration of one sort or another. You should get a popup when the integration is added. Do you have a popup blocker that might be blocking it?

**Reply by plyusnin** [reply]
Thank you for your answer! I don’t get any pop up. Should it be a pop up window or something inside of the page? I disabled the ad-blocker and also added the HA to the white list, it didn’t help. I also tried it in Safari and Chrome without success. Tried in the mobile safari and the iOS app — nothing…

**Reply by ekutner** [reply]
It should be a regular popup which takes to the the Home Connect login and authorization process. I can't say why it doesn't open for you other than saying with very high confidence that it is not an integration issue. The things I can think of: double check you configuration, make sure you are connecting directly to the HA instance and not though some proxy like Nanu Casa, try removing the integration completely from HACS, then reinstall it and then add it to HA again, try from a different computer or from you phone.

**Reply by plyusnin** [reply]
Oh man, I was so stupid! I only installed the integration in HACS but didn't get that I also need to add the integration in HA -> Settings -> Devices & Services -> Integrations 🤦‍♂️ It's even stated in the guide. Thank you for answering and thank you for the integration!
